packages/nouns-contracts/contracts/governance/data/NounsDAOData.sol

```diff
@@ -174,6 +185,10 @@ contract NounsDAOData is OwnableUpgradeable, UUPSUpgradeable, NounsDAODataEvents
             encodedProp = abi.encodePacked(proposalIdToUpdate, encodedProp);
         }
 
+        if (msg.value > 0) {
+            feeRecipient.sendValue(msg.value);
```

Comment:
nit: I think OZ doesn't add anything in this case. for me it just forces me to go into their implementation to see what it does.